pkgs/desktops/gnome-3/update.nix

```diff
@@ -0,0 +1,10 @@
+{ lib, writeScript, python3, common-updater-scripts, coreutils, gnugrep, gnused }:
+{ packageName, attrPath ? packageName, versionPolicy ? "odd-unstable" }:
+
+let
+  python = python3.withPackages (p: [ p.requests ]);
+in writeScript "update-${packageName}" ''
+PATH=${lib.makeBinPath [ common-updater-scripts coreutils gnugrep gnused python ]}
+latest_tag=$(python "${./find-latest-version.py}" "${packageName}" "${versionPolicy}" "stable")
```

Comment:
I would like to make the stability configurable but, for some reason, I cannot make `update.nix` accept arbitrary arguments:

```ShellSession
$ nix-instantiate -E 'args@{foo, ...}: builtins.trace args {}' --arg foo 5 --arg bar 6
trace: { foo = <CODE>; }
```